tools/permutate_axis/permutate_axis.xml

```diff
@@ -0,0 +1,30 @@
+<tool id="permutate_axis" name="Permutate Axis" version="0.1">
+  <description> Permutates axes <description/>
+  <requirements>
+    <requirement type="package" version="0.20.0" >scikit-image</requirement>
+    <requirement type="package" version="1.15.4">numpy</requirement>
+  <requirements/>
+  <command>
+    <![CDATA[
+    python "$__tool_directory__/permutate_axis.py" '$input' '$output' --axis $axis $permutate
```

Comment:
single quote

Reply:
@bgruening which elements are shared according to your observation? To my point of view just the requirements are sometimes the same, or the citation.